src/components/Executions.js

```diff
@@ -179,60 +370,69 @@ class Executions extends Component {
 
     return (
       <Main>
-        <div className={classes.container}>
-          <div className={classes.pythonLogo}>
-            <PythonLogo />
+        <Container>
+          <h1>{t('Executions')}</h1>
```

Comment:
I prefer increasing the margin. JC gave us feedback the other day that it would be more consistent if we had an add button as for datasets and algorithms to create new executions. So this will eventually change later.